### PR TITLE
Convert existing tables to use the simpler 3d table definition style in rusefi_config.txt

### LINKED ISSUE #3465: leverage #3464 for existing tables
@racer-coder can you please leverage #3464 for some of the current tables like
```
custom map_estimate_table_t	2*@@FUEL_RPM_COUNT@@x@@FUEL_LOAD_COUNT@@ array, U16, @OFFSET@, [@@FUEL_RPM_COUNT@@x@@FUEL_LOAD_COUNT@@], "kPa", {1/@@PACK_MULT_MAP_ESTIMATE@@}, 0, 0, 100, 2
custom ve_table_t 4*@@FUEL_RPM_COUNT@@x@@FUEL_LOAD_COUNT@@ array, F32, @OFFSET@, [@@FUEL_RPM_COUNT@@x@@FUEL_LOAD_COUNT@@], "%", 1, 0, 0, 999, 2
custom lambda_table_t @@FUEL_RPM_COUNT@@x@@FUEL_LOAD_COUNT@@ array, U08, @OFFSET@, [@@FUEL_RPM_COUNT@@x@@FUEL_LOAD_COUNT@@], "deg", {1/@@PACK_MULT_LAMBDA_CFG@@}, 0, 0.6, 1.5, 2
custom afr_table_t @@FUEL_RPM_COUNT@@x@@FUEL_LOAD_COUNT@@ array, U08, @OFFSET@, [@@FUEL_RPM_COUNT@@x@@FUEL_LOAD_COUNT@@], "deg", {1/@@PACK_MULT_AFR_CFG@@}, 0, 0, 25, 1

```

### PR DESCRIPTION
**Pull request by racer-coder**
This fixes #3465.

### CODE REVIEW

**Comment by rusefillc** [comment]
Niiiiiiiiiiiiiiiiiiiiiice!

**Comment by rusefillc** [comment]
There is that https://rusefi.com/forum/viewtopic.php?f=13&t=325 thing...

File: firmware/controllers/algo/rusefi_types.h [hunk 1]
```diff
@@ -23,6 +23,10 @@ typedef unsigned int time_t;
 #include "rusefi_enums.h"
 #include "firing_order.h"
 
+#if __cplusplus
```
Comment (mck1117) [inline firmware/controllers/algo/rusefi_types.h:26]: do we still actually include this file from a `.c` file?

Comment (racer-coder) [reply firmware/controllers/algo/rusefi_types.h:26]: yeah, it gets included in the precompiled headers which are used by main.c and os_util.c, among others.

Comment (mck1117) [reply firmware/controllers/algo/rusefi_types.h:26]: image

Comment (mck1117) [reply firmware/controllers/algo/rusefi_types.h:26]: oh, but there are other headers that include `rusefi_types.h` that are included from `.c`

**Pull request merged**